Dockerfile

```diff
@@ -7,7 +7,7 @@ LABEL org.label-schema.vcs-url="https://github.com/Dwolla/atlassian-crowd-docker
 EXPOSE 8443
 
 RUN apk add --upgrade apk-tools && \
-    apk add -U jq openssl python py-pip tomcat-native && \
+    apk add -U jq openssl python py-pip && \
```

Comment:
So what installs Tomcat now?

Reply:
`tomcat-native` and `tomcat` are two separate things, with `tomcat-native` being installed for 3.3.0 as I saw:

```
INFO [main] org.apache.catalina.core.AprLifecycleListener.lifecycleEvent The APR based Apache Tomcat Native library which allows optimal performance in production environments was not
 found on the java.library.path:
```

When setting up crowd.  It breaks the "run anywhere" of java by using native tools to speed up SSL.

tomcat itself is deployed ( and was updated since last built, which is why we're running into this issue to begin with) upstream from the crowd tarball:

https://github.com/teamatldocker/crowd/blob/master/Dockerfile#L39-L45

 ```
➭ tar -tvf atlassian-crowd-3.3.5.tar.gz |grep bin | head
-rw-rw-r--  0 bamboo-agent bamboo-agent  309 May  2 08:00 atlassian-crowd-3.3.5/apache-tomcat/bin/setenv.bat
-rwxr--r--  0 bamboo-agent bamboo-agent  370 May  2 08:00 atlassian-crowd-3.3.5/apache-tomcat/bin/setenv.sh
-rwxr--r--  0 bamboo-agent bamboo-agent 3680 Apr 10 10:31 atlassian-crowd-3.3.5/apache-tomcat/bin/setclasspath.sh
-rwxr--r--  0 bamboo-agent bamboo-agent 1902 Apr 10 10:31 atlassian-crowd-3.3.5/apache-tomcat/bin/shutdown.sh
-rwxr--r--  0 bamboo-agent bamboo-agent 1904 Apr 10 10:31 atlassian-crowd-3.3.5/apache-tomcat/bin/startup.sh
-rwxr--r--  0 bamboo-agent bamboo-agent 5515 Apr 10 10:31 atlassian-crowd-3.3.5/apache-tomcat/bin/tool-wrapper.sh
-rwxr--r--  0 bamboo-agent bamboo-agent 1922 Apr 10 10:31 atlassian-crowd-3.3.5/apache-tomcat/bin/configtest.sh
-rwxr--r--  0 bamboo-agent bamboo-agent 1908 Apr 10 10:31 atlassian-crowd-3.3.5/apache-tomcat/bin/version.sh
-rwxr--r--  0 bamboo-agent bamboo-agent 1965 Apr 10 10:31 atlassian-crowd-3.3.5/apache-tomcat/bin/digest.sh
-rwxr--r--  0 bamboo-agent bamboo-agent 23463 Apr 10 10:31 atlassian-crowd-3.3.5/apache-tomcat/bin/catalina.sh
```